tensorflow/python/keras/backend.py

```diff
@@ -2759,7 +2759,8 @@ class Function(object):
       outputs: Output tensors to fetch.
       updates: Additional update ops to be run at function call.
       name: A name to help users identify what this function does.
-      session_kwargs: Arguments to `tf.Session.run()`: `fetches`, `feed_dict`.
+      session_kwargs: Arguments to `tf.Session.run()`:
+                      `fetches`, `feed_dict`, `options`.
```

Comment:
There is `run_metadata` as well, which is useful for things like getting the partition graphs and `step_stats`.